modules/venat-module-market/src/data/universalis.ts

```diff
@@ -0,0 +1,32 @@
+import axios, { AxiosResponse } from 'axios';
+import { LookupResult } from './result';
+
+export interface UniversalisMarketInfo {
```

Comment:
Do you want to show any of the average stats or would that be a later change/separate command?

Reply:
Later change, I think this is the MVP for this module.